src/components/ContentUploader/UploadState.js

```diff
@@ -47,16 +48,23 @@ const UploadState = ({ canDrop, hasItems, isOver, isTouch, view, onSelect, isFol
                     />
                 ) : (
                     <UploadStateContent
-                        fileInputLabel={<FormattedMessage {...messages.uploadEmptyFileInput} />}
+                        fileInputLabel={
+                            !Browser.isIE() ? (
```

Comment:
Nit: I would make this be the opposite. Browser.isIE() ? doIeThings : doOtherThings